packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/Title.tsx

```diff
@@ -0,0 +1,84 @@
+import { getSandboxName } from '@codesandbox/common/lib/utils/get-sandbox-name';
+import { ESC, ENTER } from '@codesandbox/common/lib/utils/keycodes';
+import React, { ChangeEvent, KeyboardEvent, useState } from 'react';
+
+import { useOvermind } from 'app/overmind';
+
+import { Text, Input, Stack } from '@codesandbox/components/lib';
+import styled from 'styled-components';
+import { PenIcon } from './icons';
+
+const Icon = styled(PenIcon)`
+  cursor: pointer;
+  display: none;
+`;
+
+const Name = styled(Stack)`
+  :hover {
+    svg {
+      display: block;
+    }
+  }
+`;
```

Comment:
object notation please?

Reply:
We agreeed that it would be for the components, I want my strings :(